### PR TITLE
Metrics RC announcement blog post

### PR DESCRIPTION
**Pull request by mtwo**
This blog post will go live at 9:00 AM PT on Wednesday, May 18th

---

Preview: https://deploy-preview-1339--opentelemetry.netlify.app/blog/2022/metrics-announcement/

### CODE REVIEW

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:29]:
```suggestion
If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, or Collector, then you can access GA metrics functionality by updating your OpenTelemetry artifacts to their latest versions. You can find documentation and getting started guides for each artifact’s metrics capabilities within [the official OpenTelemetry documentation](../../../docs) and examples and supplementary documentation is often available within each artifact’s corresponding GitHub repository.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (austinlparker) [inline content/en/blog/2022/metrics-announcement.md:32]:
```suggestion
Distributed traces and logs were the two halves of OpenTelemetry’s core promise when we announced it at Kubecon EU in 2019. With the general availability of metrics, we have produced the capabilities that we originally set out to create, meaning that we can shift our focus to further investing into the robustness and ease of use of each component, the number of data sources that OpenTelemetry can capture telemetry from (either through the OpenTelemetry APIs, OTLP, or otherwise), and new capabilities and signal types.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:14]:
```suggestion
    - People who develop and maintain web services or client applications, so that they can produce custom metrics or interact with existing metrics. For example, an e-commerce company could use the API to track how many purchases are made over time.
    
```

File: content/en/blog/2022/metrics-announcement.md
Comment (cartermp) [inline content/en/blog/2022/metrics-announcement.md:12]: This is a little confusing coming before the next major point, which indicates that there are no GA-declared languages. The examples given still ultimately end up depending on a given language's support, I think?

File: content/en/blog/2022/metrics-announcement.md
Comment (cartermp) [inline content/en/blog/2022/metrics-announcement.md:30]: "You can find documentation [...]"

image

Would it be helpful to say that documentation is in-progress? Because, uhh, we're still working through pretty major tracing docs, let alone metrics 😬

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:12]: Good point. The APIs are mostly GA while the SDKs are not (even for a given language), but I don't think that I got this across very clearly. Fixing.

Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:12]:
```suggestion
- Metrics functionality included in the OpenTelemetry language-specific APIs, which provide language-specific interfaces that can create and manipulate metrics, and associate metadata and attributes to each. These are useful for:
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:15]:
```suggestion
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:14]:
```suggestion
    - People who develop and maintain web services or client applications, so that they can produce custom metrics or interact with existing metrics. For example, an e-commerce company could use the API to track how many purchases are made over time.
    
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:17]:
```suggestion
    - Developers of applications within a technology organization, who want to capture metrics generated by OpenTelemetry APIs within their applications, either by their own developers or from shared libraries that their applications depend on. These metrics can be exported through OTLP or exported through any other OpenTelemetry exporter.
    
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:22]:
```suggestion
    - Anyone who wants to convert metrics from one format to another. For example, the Collector can capture metrics from a mix of OTLP and Prometheus sources, and then send all of these to a single destination using OTLP (with the standard OpenTelemetry semantic conventions), Prometheus, or any other exporter.
    
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:32]: :)

```suggestion
If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, or Collector, then you can access GA metrics functionality by updating your OpenTelemetry artifacts to their latest versions. We're currently producing documentation and getting started guides for each artifact’s metrics capabilities within [the official OpenTelemetry documentation](../../../docs). Examples and supplementary documentation are also being added to each artifact’s corresponding GitHub repository.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:16]:
```suggestion
- Metrics functionality included in the OpenTelemetry SDKs for Java, .Net, Python, and JS (coming next week) SDKs, which capture metrics from the APIs and perform some amount of processing. Metrics support for other languages is still in development. These are useful for:
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:29]:
```suggestion
All of this functionality is additive to OpenTelemetry’s existing tracing support, and both signal types share the same metadata and semantic conventions. As of this announcement, the Java, .Net, and Python SDKs have issued release candidates for OpenTelemetry’s metric capabilities, JS is coming next week, and more languages will be issuing metrics release candidates throughout the coming months. Each of these releases will be followed by general availability after we receive feedback from users.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (chalin) [inline content/en/blog/2022/metrics-announcement.md:7]: Editorial: As a general comment, please consider line-wrapping at 80 chars.

File: content/en/blog/2022/metrics-announcement.md
Comment (chalin) [inline content/en/blog/2022/metrics-announcement.md:8]: > OpenTelemetry’s metrics capabilities are now available as release candidates!

It isn't for all languages, so it might be better to be clear about this from the start. WDYT @mtwo?

At a minimum, consider something like this:

> OpenTelemetry’s ... as release candidates (RCs) -- for most languages, for details, read on.

Or list the languages that are in RC.

File: content/en/blog/2022/metrics-announcement.md
Comment (chalin) [inline content/en/blog/2022/metrics-announcement.md:29]: > As of this announcement, the Java, .Net, and Python SDKs have issued release candidates for OpenTelemetry’s metric capabilities, JS is coming next week, and more languages will be issuing metrics release candidates throughout the coming months.

Consider making this more visually distinctive. E.g.:

> As of this announcement, the following languages have issued release candidates:
>
> - [Java](...)
> - [.Net](...)
> - [Python](...)
>
> [JavaScript](...) release is planned for next week, and more ... months.

File: content/en/blog/2022/metrics-announcement.md
Comment (chalin) [inline content/en/blog/2022/metrics-announcement.md:31]: Nite: add a space between the heading and the text that follows (here and below).

File: content/en/blog/2022/metrics-announcement.md
Comment (chalin) [inline content/en/blog/2022/metrics-announcement.md:32]: (Another reason why line-wrapping is good: it then becomes easier to pin specific review comments. In the meantime, I'll quote the text in the following series of comments on this paragraph.)

> or Collector

-->

> and Collector

Comment (chalin) [reply content/en/blog/2022/metrics-announcement.md:32]: > you can access GA metrics

GA or RC?

Comment (chalin) [reply content/en/blog/2022/metrics-announcement.md:32]: Replace `../../../docs` with just `/docs/`

Comment (chalin) [reply content/en/blog/2022/metrics-announcement.md:32]: Consider changing:

> We're currently producing documentation ...

to

> We're currently updating [the official OpenTelemetry documentation](/docs/) for each artifact’s metrics capabilities.

**Review by chalin — COMMENTED** [review]
See inline comments.

As I mentioned in Slack, once these changes are addressed, we can merge right away, and on the official drop date/time republish without the `draft` status.

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:1]: I agree, but unfortunately this will impact the URL for the post on opentelemetry.io, which has already been direct linked by many of the vendors that are cross-posting this

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:8]:
```suggestion
OpenTelemetry’s metrics capabilities are now available as release candidates, starting with Java, .Net, and Python! This means that the specification, APIs, SDKs, and other components that author, capture, process, and otherwise interact with metrics now have the full set of OpenTelemetry metrics functionality and are ready for use. These release candidates will be promoted to general availability throughout the next few weeks.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:14]:
```suggestion
    - Developers who create and maintain web services or client applications, so that they can produce custom metrics or interact with existing metrics. For example, an e-commerce company could use the API to track how many purchases are made over time.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:32]:
```suggestion
If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, and Collector, then you can access the release candidate metrics functionality by updating your OpenTelemetry artifacts to their latest versions. We're currently updating [the official OpenTelemetry documentation](/docs) for each artifact's metrics capabilities. Examples and supplementary documentation are also being added to each artifact’s corresponding GitHub repository.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:7]: Will do on the next one. This one was imported from Google docs and is going live tomorrow, so I'd prefer not to mess with it :)

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:29]:
```suggestion
All of this functionality is additive to OpenTelemetry’s existing tracing support, and both signal types share the same metadata and semantic conventions. As of this announcement, the following languages have issued metrics release candidates:

- Java
- .Net
- Python

The RC release for JS is planned for next week, and more languages will be issuing metrics release candidates throughout the coming months. Each of these releases will be followed by general availability after we receive feedback from users.
```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [reply content/en/blog/2022/metrics-announcement.md:31]:
```suggestion
## Getting Started

```

File: content/en/blog/2022/metrics-announcement.md
Comment (mtwo) [inline content/en/blog/2022/metrics-announcement.md:34]:
```suggestion
## What’s Next for OpenTelemetry

```

**Review by alolita — CHANGES_REQUESTED** [review]
Please make requested changes to Collector section highlighting key features and interoperability support.

**Comment by mtwo** [comment]
Alolita's changes are merged into the PR, so I'm going to merge this into main

**Pull request merged**
